run-clang-format.py

```diff
@@ -25,20 +25,34 @@
 style = sys.argv[4]
 print("Style: " + style)
 
-def runclangformat(filepath):
-    print("Checking: " + filepath)
+class FileInfo:
+    def __init__(self, file_path, parent_dir):
+        self.file_path = file_path
+        self.parent_dir = parent_dir
+
+
+def runclangformat(fileInfo):
+    filepath = fileInfo.file_path
+    if fileInfo.parent_dir:
+        print("Checking: " + fileInfo.parent_dir)
```

Comment:
This looks really weird! And took longer for me to figure out than needed... he he

There's no need for this complexity. Just move the `print("Checking: " + unique_dir)` into the `collectfiles` function, and print when a unique dir is detected.

Reply:
Argh! You merged it!